### DISCUSSION TITLE
Apollo部署在k8s1.18中，使用apollo用户登录后没有管理员工具

### DISCUSSION THREAD

**Discussion by ghl1024** [apolloconfig/apollo#3842] [Q&A] [answered]
各个组件日志正常，登录后没有管理员工具，什么都做不了
![image](https://user-images.githubusercontent.com/45088633/126903223-8c3d314c-a591-4fce-96e1-5f490e966baf.png)

**Comment by nobodyiam** [comment, marked as answer]
看一下[3.1.4 superAdmin - Portal超级管理员](https://www.apolloconfig.com/#/zh/deployment/distributed-deployment-guide?id=_314-superadmin-portal%e8%b6%85%e7%ba%a7%e7%ae%a1%e7%90%86%e5%91%98)配置

**Reply by ghl1024** [reply]
这个问题暂时通过自建数据库解决了，由于之前使用的是腾讯云的数据库，在DMC中执行一直不生效，联系售后说只能在外面导入，还有什么数据库小版本之类的问题，但是没有任何地方给用户一些提示，售后说这个问题已经反馈给产品了。
